### PR TITLE
add: Generic system temperature support

### PR DESCRIPTION
**Pull request by locnnil**
* Complements #149
* Use `lm-sensors` package to read temperature
* Detect Raspberry Pi and keep using vcgencmd in case.

### CODE REVIEW

File: scripts/dracula.sh [hunk 1]
```diff
@@ -298,9 +298,9 @@ main()
       continue
     fi
 
-    if [ $plugin = "rpi-temp" ]; then
+    if [ $plugin = "sys-temp" ]; then
       IFS=' ' read -r -a colors <<< $(get_tmux_option "@dracula-rpi-temp-colors" "green dark_gray")
```
Comment (locnnil) [inline scripts/dracula.sh:302]: Does it have to change as well?

Comment (ethancedwards8) [reply scripts/dracula.sh:302]: Yes. It should in order to keep things consistent.

Comment (fvincenzo) [reply scripts/dracula.sh:302]: Agreed, it should be `dracula-sys-temp-colors`.
@ethancedwards8  Do we have a place where all the parameters are documented? I am struggling to keep up with the latest and greatest in my `tmux.conf` file.

File: scripts/sys_temp.sh [hunk 2]
```diff
@@ -3,7 +3,13 @@
 export LC_ALL=en_US.UTF-8
 
 get_temp() {
-  echo "$(vcgencmd measure_temp | sed 's/temp=//')"
+  if [[ -d "/proc/device-tree/" && -f "/proc/device-tree/model" && $(grep -i 'Raspberry' /proc/device-tree/model) ]]; then
+    # It's a Raspberry pi
+    echo "$(vcgencmd measure_temp | sed 's/temp=//')"
+  else
+    # It's a Regular pc
```
Comment (fvincenzo) [inline scripts/sys_temp.sh:10]: How about I have linux on a Mac? 😄 
I think it should be "It's anything else" or just remove the comment. Your choice.

**Review by fvincenzo — COMMENTED** [review]
Nice patch. Thanks!

Just few nits from me.

File: scripts/dracula.sh (hunk 1 above)
Comment (locnnil) [reply scripts/dracula.sh]: Addressed on https://github.com/dracula/tmux/pull/307/commits/8efdc955df0d6dabca42d65c9bd5fe69a2651379. Thank you!

File: README.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ Configuration and options can be found at [draculatheme.com/tmux](https://dracul
     set -g @dracula-show-battery-status true
     ```
 - If forecast information is available, a ☀, ☁, ☂, or ❄ unicode character corresponding with the forecast is displayed alongside the temperature
-- System temperature on Raspberry PI
+- System temperature on Raspberry PI and on regular computers
```
[the remark this answers is not in the document]
Comment (locnnil) [reply README.md]: Agree! Changed to "Read system temperature" on https://github.com/dracula/tmux/pull/307/commits/8efdc955df0d6dabca42d65c9bd5fe69a2651379.

**Comment by locnnil** [comment]
@fvincenzo and @ethancedwards8 thank you very much for your reviews, I appreciate it! This is now ready to be reviewed again. 

Just one more question, about the icon, how and where is it defined? Do we already have one for system temperature?

File: scripts/dracula.sh (hunk 1 above)
Comment (ethancedwards8) [reply scripts/dracula.sh]: > Agreed, it should be `dracula-sys-temp-colors`. @ethancedwards8 Do we have a place where all the parameters are documented? I am struggling to keep up with the latest and greatest in my `tmux.conf` file.

Kinda. INSTALL.md. It isn't 100% complete though. That is something we are working on.

**Comment by ethancedwards8** [comment]
Thank you for your contribution!

**Pull request merged**